stun_server/region/ecs.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_ecs_service" "stun-server" {
```

Comment:
<https://github.com/home-assistant/deployments/pull/87/files#diff-53a630ff67f218bf174fff7b35b58537507d636383623b0088562416f92eb384R12>
Service is also defined here, so this is conflicting.